<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="2">

  <testsuite name="TestOwnWallets"
    errors="1" failures="0" hostname="" id="" package="TestOwnWallets" skipped="1"
    tests="2" time="0.01300" timestamp="Mon, 30 Jul 2018 13:35:14 GMT">
  
    <testcase name="wallets" classname="TestOwnWallets" time="0.01300" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Test Own Wallets / wallets]:

  TypeError: Cannot read property &#39;CONTROL&#39; of undefined
    at Object.wallets (/home/evg/WebstormProjects/ale-wallet/test/e2e/specs/tests/TestOwnWallets.js:25:28)
    at Module.call (/home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/lib/runner/module.js:62:34)
    at /home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/lib/runner/testcase.js:70:29
    at _fulfilled (/home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/node_modules/q/q.js:796:13)
    at /home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/node_modules/q/q.js:556:49
    at runSingle (/home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/node_modules/q/q.js:137:13)
    at flush (/home/evg/WebstormProjects/ale-wallet/node_modules/nightwatch/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    </system-err>
  

  
    
    <testcase
      name="here we check wallet logic" classname="TestOwnWallets">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
